### DISCUSSION TITLE
Not fixed width?

### DISCUSSION THREAD

**Discussion by igorjacauna** [bbc/peaks.js#502] [Q&A] [answered]
Hello, I'm trying to build a multitrack player and wish to not get fixed width for the waveforms when I stack the tracks.

The tracks are getting dealigned time markers like:

![image](https://github.com/bbc/peaks.js/assets/459127/3a2c9bfc-2b14-4eb6-beec-0b89f7bb6b74)

**Comment by chrisn** [comment, marked as answer]
It should be possible, if you use the zoomview, not the overview, as this allows you to control the zoom level.

**Reply by igorjacauna** [reply]
Thanks! It worked!
